apps/wing/src/commands/test/test.ts

```diff
@@ -357,9 +364,98 @@ async function runTestsWithRetry(
   return results;
 }
 
+function inferSeverityOfEvent(trace: std.Trace): "error" | "warn" | "info" | "debug" | "verbose" {
+  if (trace.data.status === "failure") {
+    return "error";
+  }
+  if (trace.type === TraceType.LOG) {
+    return "info";
+  }
+  if (trace.type === TraceType.RESOURCE) {
+    return "debug";
+  }
+  if (trace.type === TraceType.SIMULATOR) {
+    return "verbose";
+  }
+  return "verbose";
+}
+
+const SEVERITY_STRING = {
+  error: "ERROR  ",
+  warn: "WARNING",
+  info: "INFO   ",
+  debug: "DEBUG  ",
+  verbose: "VERBOSE",
```

Comment:
The padding looks a little strange inside the brackets, how does it look at the end? i.e. `[VERBOSE]` and `[INFO]   `

Reply:
Added